rps.js

```diff
@@ -0,0 +1,43 @@
+const playGame = function(a, b = random()) {
+    if (a===b){
+        console.log('tie')
+    }
+    else {
+        if (a==='rock' && b === 'paper'){
+            console.log('b win');
+        }
+      if (a==='rock' && b === 'scissors'){
+            console.log('a win');
+        }
+      if (a==='scissors' && b === 'paper'){
+            console.log('a win');
+        }
+      if (a==='scissors' && b === 'rock'){
+            console.log('b win');
+        }
+      if (a==='paper' && b === 'scissors'){
+            console.log('b win');
+        }
+      if (a==='paper' && b === 'rock'){
+            console.log('a win');
+        }
```

Comment:
You're making your program go through each condition even when it finds the result it wants, separating them with `else if` saves time